app/views/invites/confirm_request.html.erb

```diff
@@ -23,7 +23,7 @@
         <%= f.password_field :password_confirmation, autocomplete: t('users.password'), class: 'form-control', required: true %>
       </div>
 
-      <%= f.hidden_field :role, value: User::ROLES[0] %>
+      <%= f.hidden_field :role, value: @invite.email %>
```

Comment:
Why role has email in it? Isn't it saved before? It is not required (Discuss with me)